settings-schema.json

```diff
@@ -69,31 +23,34 @@
                     "default_nesting":{
                         "title":"Default Nesting",
                         "type":"boolean"
+                    },
+                    "role":{
+                        "title":"Specify Optional Role",
+                        "type":"string"
                     }
                 }
             },
             "default": [
                 {
-                    "settings_col": "site_col",
-                    "value_col" : null,
+                    "value_col" : "sitename",
                     "label": "Site",
-                    "default_nesting": true
+                    "default_nesting": true,
+                    "role": "site_col"
                 },
                 {
-                    "settings_col": "id_col",
-                    "value_col" : null,
+                    "value_col" : "id_col",
                     "label": "Subject ID",
-                    "default_nesting": true
+                    "default_nesting": true,
+                    "role": "id_col"
                 },
                 {
-                    "settings_col": "visit_col",
-                    "value_col" : null,
+                    "value_col" : "visit_col",
                     "label": "Folder",
-                    "default_nesting": false
+                    "default_nesting": false,
+                    "role": "visit_col"
                 },
                 {
-                    "settings_col": "form_col",
-                    "value_col" : null,
+                    "value_col" : "form_col",
                     "label": "Form",
                     "default_nesting": false
```

Comment:
why doesn't this one have a role?

Reply:
Interestingly, the form column isn't used anywhere besides summarization so it doesn't need one - as far as the chart knows it's just another variable